apps/mobile/metro.config.cjs

```diff
@@ -48,9 +48,17 @@ config.resolver.resolveRequest = (context, moduleName, platform) => {
   if (moduleName.startsWith('@/')) {
     return context.resolveRequest(context, moduleName, platform);
   }
-
-  if (moduleName === 'axios') {
-    return context.resolveRequest(context, moduleName, platform);
+  // Use the browser version of the package for React Native
```

Comment:
I made a task [LEA-2641](https://linear.app/leather-io/issue/LEA-2641/investigate-replacing-axios-with-native-fetch) to investigate just removing `axios` in future